gammapy/irf/__init__.py

```diff
@@ -58,3 +58,5 @@
         EDispMap,
     ]
 )
+
+__all__.extend(cls.__name__ for cls in IRF_REGISTRY)
```

Comment:
Was that intentionally, @Bultako ?

Shouldn't be needed anymore

Reply:
sorry, it was introduced by mistake when resolved conflicts with the master branch. 

you can safely remove it.